Cargo.toml

```diff
@@ -18,7 +18,7 @@ categories = ["no-std", "embedded"]
 include = ["/Cargo.toml", "src/**/*.rs", "/README.md", "/LICENSE"]
 
 [dependencies]
-ink_core = { version = "2", git = "https://github.com/paritytech/ink", tag = "latest-v2", package = "ink_core", default-features = false }
+ink_core = { version = "2.1", git = "https://github.com/paritytech/ink", tag = "latest-v2", package = "ink_core", default-features = false }
```

Comment:
You're getting the code from a specific tag/commit `tag = "latest-v2"`. The version number change won't have any impact until you point to a different commit.

Reply:
hmm, I have no idea what happened. Actually I've just met the error shown in https://github.com/paritytech/ink-types-node-runtime/issues/12  and fixed it by adding `version = '2.1'`